app/src/main/java/com/dcv/movie_demo/screen/adapter/MenuAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package com.dcv.movie_demo.screen.adapter
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.dcv.movie_demo.screen.favorite.FavoriteFragment
+import com.dcv.movie_demo.screen.genres.GenresFragment
+import com.dcv.movie_demo.screen.trending.TrendingFragment
+
+class MenuAdapter(fragmentActivity: FragmentActivity): FragmentStateAdapter(fragmentActivity) {
+
+    private val fg = mutableListOf<Fragment>()
```

Comment:
Đặt tên phải rõ ràng nhé
```suggestion
    private val fragments = mutableListOf<Fragment>()
```